.github/workflows/docs.yml

```diff
@@ -10,19 +10,19 @@ jobs:
   docs:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: borales/actions-yarn@v2.0.0
+      - uses: actions/checkout@v1
       - name: Set up Python
         uses: actions/setup-python@v1
         with:
           python-version: "3.8"
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12'
```

Comment:
I am not sure, do we use this action here? It is similar in `build-docs.yml`

Reply:
I guess its just copy pasta from python version. I will check and remove if not needed